adhocracy4/projects/migrations/0051_migrate_point_field.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.17 on 2025-02-19 15:49
+
+from django.db import migrations
+
+
+def migrate_project_geos_point_field(apps, schema_editor):
+    project = apps.get_model("a4projects", "Project")
+    for project in project.objects.all():
+        project.point = project.geos_point
```

Comment:
Shouldn't it also has a check if `project.geos_point` exists?

Reply:
added